src/chrome/content/rules/Uni.edu.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="UNI">
```

Comment:
This name is to generic. Please use something like "University of Northern Iowa" or "uni.edu".